hoot-core/src/main/cpp/hoot/core/conflate/poi-polygon/PoiPolygonReviewReducer.cpp

```diff
@@ -703,16 +703,20 @@ bool PoiPolygonReviewReducer::_poiNeighborIsCloserToPolyThanPoi(ConstElementPtr
         {
           LOG_VART(poi);
           LOG_VART(poiNeighbor);
+          //  This only handles ways as polygons and not relations
```

Comment:
There are cases where the poly coming in is a relation and this was causing a crash using the data I was working with.  This makes sure that only valid ways will be processed.

Reply:
I wouldn't mind eventually adding relation support here. Low priority, though.